com.unity.render-pipelines.universal/Runtime/UniversalRenderer.cs

```diff
@@ -198,12 +198,12 @@ public UniversalRenderer(UniversalRendererData data) : base(data)
             m_DepthNormalPrepass = new DepthNormalOnlyPass(RenderPassEvent.BeforeRenderingPrePasses, RenderQueueRange.opaque, data.opaqueLayerMask);
             m_MotionVectorPass = new MotionVectorRenderPass(m_CameraMotionVecMaterial, m_ObjectMotionVecMaterial);
 
-            if (this.renderingMode == RenderingMode.Forward)
+            if (this.renderingModeRequested == RenderingMode.Forward)
             {
                 m_PrimedDepthCopyPass = new CopyDepthPass(RenderPassEvent.AfterRenderingPrePasses, m_CopyDepthMaterial);
             }
 
-            if (this.renderingMode == RenderingMode.Deferred)
+            if (this.renderingModeRequested == RenderingMode.Deferred)
```

Comment:
Why these use renderingModeRequested? Shouldn't these depend on the renderingModeActual?

Reply:
@sandy-carter-unity this is a good point. It seems sensible that the actual rendering mode is figured out very early before all of these settings. 

Actually, thinking about it a little bit now, we should expose a property to get the rendering mode that always return the actual one and the requested / setting is private setting, this makes it less confusing.

Reply:
Is this what you mean?
```csharp
private RenderingMode renderingModeRequested => m_RenderingMode;
internal RenderingMode renderingModeActual => ...
```

Reply:
At that point, we could remove the `renderingModeRequested` property entirely

Reply:
Yeah, ok I see the `renderingModeRequested` is private now, but that feels useless tbh as you can just have `m_RenderingMode`?